app/workers/import_worker.rb

```diff
@@ -71,4 +73,18 @@ def process_follows
       end
     end
   end
+
+  def process_favourites
+    import_rows.each do |row|
+      begin
+        domain, account, id = row
+
+        status = (domain == Rails.configuration.x.local_domain) ? Status.find(id) : FetchRemoteStatusService.new.call("http://#{domain}/users/#{account}/#{id}.atom")
```

Comment:
Keep in mind the reason we don't have backfilling of old statuses en masse is because we don't have #1059, so if you ran an import like this on 10k statuses from e.g. ~3 months ago all public timelines would be filled with 10k statuses from 3 months ago.